proto/parca/query/v1alpha1/query.proto

```diff
@@ -333,6 +340,11 @@ message QueryResponse{
     }
 }
 
+// QueryPprofResponse holds a pb.gz pprof profile for downloading
+message QueryPprofResponse {
```

Comment:
The `Query` endpoint already has the ability to request the kind of response the user is requesting. I expected pprof to be just a different option there next to flamegraph/callgraph/top etc.

Reply:
Oh, yes. That would make a lot of sense. :smile: :100: